Lawrence/BANKaccount2.rb

```diff
@@ -0,0 +1,76 @@
+
+class Bankaccount
+  @@totalaccounts = 0
+
+  def initialize()
+      @checkbalance = 0
+      @savebalance = 0
+      @accountnumber = accountnumbercreate
+      @@totalaccounts +=  1
+  end
+
+  def displayCheckBalance
+      puts  @checkbalance
+      return @checkbalance
```

Comment:
no need for returns! ruby automatically returns the last expression so @checkbalance is okay